packages/openactive-integration-tests/test-data-generator/README.md

```diff
@@ -13,10 +19,12 @@ npm run test-data-generator # all active features specified in the current confi
 
 npm run test-data-generator -- common-error-conditions # only enough data for the common-error-conditions feature
 
-npm run test-data-generator -- --output ./test-data.json # output to ./test-data.json
+npm run test-data-generator -- --output-dir ../../../test-data-generator-output # output to ../../../test-data-generator-output
```

Comment:
`--output-dir` seems a bit simpler to maintain when we're now dealing with multiple output files and I expect sufficiently customizable for most purposes? Please correct me if I'm wrong!

Reply:
Yep makes sense